src/nidcpower/metadata/attributes.py

```diff
@@ -80,7 +73,6 @@
     },
     1050327: {
         'access': 'read only',
-        'channel_based': False,
```

Comment:
Is the nimi-python metadata expansion defaulting lack of `supported_rep_caps` to mean `channels`?
I think that's a bad idea. We should make it so that the nimi-python metadata is explicit. 
Lack of `supported_rep_caps` should mean "None".

Reply:
> Is the nimi-python metadata expansion defaulting lack of supported_rep_caps to mean channels?

I don't think this is true, but I'll let Luis confirm.

Reply:
Shreyas is correct. That's not true. The lack of `supported_rep_caps` means "None". That's why all those `'channel_based': False` attributes simply disappeared without having to be replaced by anything.